src/NodeJS/Server.php

```diff
@@ -46,6 +46,11 @@
     protected $threshold;
 
     /**
+     * @var array
+     */
+    protected $options;
```

Comment:
@berliner , please make this change. I guess that's last thing before this PR is merged.